sdks/jreleaser-java-sdk-commons/src/main/java/org/jreleaser/sdk/commons/RestAPIException.java

```diff
@@ -40,6 +40,7 @@ public RestAPIException(int status, String reason) {
     }
 
     public RestAPIException(Request request, int status, String reason, Map<String, Collection<String>> headers) {
+        super(status + ": " + reason);
```

Comment:
I am not sure if reason doesn't already include the status code too.

FWIW, it wasn't practical to verify that. So you might see the status code twice, which is still much better than not seeing it at all.